Use versionless features (#175)
OpenLiberty/guide-microprofile-istio-retry-fallback: README.adoc

[source, role="no_copy"]
----
5
----

> The above command returns 5, because there was a total of 5 requests made to the `system` service that included the initial request, 3 MicroProfile retry requests, and the succesful request that you made before you paused the `system` service.

Reviewer: ```suggestion
This command returns 5 because you made a total of 5 requests to the `system` service: the initial request, 3 MicroProfile retry requests, and the successful request that you made before you paused the `system` service.
```

[source, role="no_copy"]
----
20
----

InventoryResource.java
[source, Java, linenums, role='code_column hide_tags=copyright,fallback,fallbackMethod']
----
include::finish/inventory/src/main/java/io/openliberty/guides/inventory/InventoryResource.java[]
----

> The above command returns a value of 20, indicating that a total of 20 requests are made to the `system` service that are multiplied by the following:

Reviewer: Im not sure what is meant by " a total of 20 requests are made to the `system` service that are multiplied by the following:"

does it mean 5 Istio requests x 4 MP requests = 20?
or 20 x 4 x 5 = 400?

Author: it means -> 5 Istio requests x 4 MP requests = 20

> The above command returns a value of 20, indicating that a total of 20 requests are made to the `system` service that are multiplied by the following:

Reviewer: ```suggestion
This command returns a value of 20, indicating that a total of 20 requests were made to the `system` service, including:
```

> * 5 Istio requests that included the initial request and 4 retry requests that you enabled in the [hotspot=attempts file=0]`traffic.yaml` file

Reviewer: ```suggestion
* 5 Istio requests: the initial request and 4 retry requests that you enabled in the [hotspot=attempts file=0]`traffic.yaml` file
```

> * 4 requests sent by MicroProfile that included the initial request and [hotspot=mpRetry file=1]`3` retry requests

Reviewer: ```suggestion
* For each of the 5 Istio requests, MicroProfile sent 4 requests:  the initial request and [hotspot=mpRetry file=1]`3` retry requests
```

[source, role="no_copy"]
----
5
----

> The above command returns 5, indicating that a total of 5 requests are made to the `system` service.

Reviewer: ```suggestion
This command returns 5, indicating that a total of 5 requests were made to the `system` service.
```

[source, role="no_copy"]
----
1
----

> The above command returns 1, indicating that totally 1 request is made to the `system` service.

Reviewer: ```suggestion
This command returns 1, indicating that 1 request was made to the `system` service.
```

The Istio retries that you enabled on the `inventory` service are not performed.

> Although the [hotspot=getPropertiesForHost file=1]`getPropertiesForHost()` method failed, the [hotspot=fallbackMethod file=1]`getPropertiesFallback()` fallback method is called.

Reviewer: ```suggestion
Although the [hotspot=getPropertiesForHost file=1]`getPropertiesForHost()` method failed, the [hotspot=fallbackMethod file=1]`getPropertiesFallback()` fallback method was called.
```
